### PR TITLE
feat: add invoices method

### PR DESCRIPTION
**Pull request by pavanjoshi914**
add invoices method to fetch all the transactions

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -113,6 +113,20 @@ export class Client {
     });
   }
 
+  invoices(
+    params: {},
```
Comment (bumi) [inline src/client.ts:117]: as params we can pass in filters. do those need some type defenition?

Comment (pavanjoshi914) [reply src/client.ts:117]: what kind of filters we support? passing limit of invoices that can be retrieved as params ?

Comment (reneaaron) [reply src/client.ts:117]: This is the list of supported params: 

https://guides.getalby.com/alby-wallet-api/reference/api-reference/invoices

image

Comment (pavanjoshi914) [reply src/client.ts:117]: image

cool! but we didn't added support for params in api docs. we shall update that as well if this endpoint supports params

Comment (bumi) [reply src/client.ts:117]: we should update those, yes. can you do it? copy it from the other endpoint.

Comment (reneaaron) [reply src/client.ts:117]: I've updated gitbook: 

https://guides.getalby.com/alby-wallet-api/reference/api-reference/invoices#get-all-invoice-history

Comment (reneaaron) [reply src/client.ts:117]: I've also added some typed params for the invoices endpoint now.

**Review by rolznz — APPROVED** [review]
utACK

**Comment by pavanjoshi914** [comment]
thanks @reneaaron for the params stuff! merging this

**Pull request merged**